visual-testing-app/src/App.js

```diff
@@ -44,7 +52,7 @@ class App extends React.Component {
             <Route
               key={Component.routePath}
               path={Component.routePath}
-              component={Component.component}
+              render={() => <Component.component theme={darkTheme} />}
```

Comment:
```suggestion
              render={() => <Component.component themes={{darkTheme: darkTheme}} />}
```
?

Reply:
Do you plan to test multiple themes?

Reply:
Maybe. Either way is fine for me. I also have an idea to use the themes in `Spec` instead of inverted

Reply:
So in that case it would be nice to have a theme around the whole App, and then the specific ones around the inverted tests.

Reply:
I'm not sure I follow you... 🤔